documentation/book/ref-list-of-kafka-cluster-resources.adoc

```diff
@@ -10,6 +10,10 @@ The following resources will created by the Cluster Operator in the {ProductPlat
 `_<cluster-name>_-kafka`:: StatefulSet which is in charge of managing the Kafka broker pods.
 `_<cluster-name>_-kafka-brokers`:: Service needed to have DNS resolve the Kafka broker pods IP addresses directly.
 `_<cluster-name>_-kafka-bootstrap`:: Service can be used as bootstrap servers for Kafka clients.
+`_<cluster-name>_-kafka-external-bootstrap`:: Bootstrap service for clients connecting from outside of the {ProductPlatformName} cluster. This resource will be created only when external listener is enabled.
+`_<cluster-name>_-kafka-_<pod-id>_`:: Service used to route traffic from outside of the {ProductPlatformName} cluster to individual pods. This resource will be created only when external listener is enabled.
+`_<cluster-name>_-kafka-external-bootstrap`:: Bootstrap route for clients connecting from outside of the {ProductPlatformName} cluster. This resource will be created only when external listener is enabled and set to type `route`.
+`_<cluster-name>_-kafka-_<pod-id>_`:: Route for traffic from outside of the {ProductPlatformName} cluster to individual pods. This resource will be created only when external listener is enabled and set to type `route`.
```

Comment:
My 2 cents here. We have resources with same names here but different meaning. Instead of repeating them twice we could have just one and then adding "if the used listener is of type route then it is ...... " or we should try to have different names (maybe adding "route" suffix).

Reply:
TBH ... I can see what you mean ... but I'm personally not a big fan of either suggestion:
1. I think that would work fine if the routes and services were always created together. But they are not. The routes are only for `route` type external listener. The services for all of them. So leaving it as single entry might be confusing.
2. I don't think we should include it into the name. Especially since for many resources this is part of their DNS name later (pods, services, routes, statefulsets).

What if we change the structure of the list to be based on components? Such as:
* StatefulSets:
  * xxx
  * yyy
* Services
  * xxx
  * yyy